src/coredump/coredump.c

```diff
@@ -1212,8 +1223,20 @@ static int process_kernel(int argc, char* argv[]) {
 
         if (context.is_journald || context.is_pid1)
                 r = submit_coredump(&context, iovw, STDIN_FILENO);
-        else
+        else {
+                _cleanup_free_ char *process_name = NULL;
+                process_name = (char *)context.meta[META_EXE];
```

Comment:
uh? given you use `_cleanup_free_` above this will corrupt the memory in the contetx object when the block is left